docs/runtime-compliance-testing.md

```diff
@@ -6,4 +6,375 @@ In order to be tested for [compliance][], runtimes MUST support at least one of
+validation/process.t ................................ 20/26
+  Skipped: 6
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+
+validation/linux_cgroups_relative_pids.t .............. 3/3
+validation/hooks.t .................................... 0/1
+  Skipped: 1
+    validation/hooks.t
+
+validation/linux_rootfs_propagation_unbindable.t .... 19/20
+  not ok rootfs propagation
+    error: 'rootfs expected to be unbindable, but not'
+
+validation/linux_cgroups_relative_cpus.t .............. 7/7
+validation/misc_props.t ............................... 2/3 4s
+  not ok runtimes that are reading or processing this configuration file MUST generate an error when invalid or unsupported values are encountered
+    reference: >-
+      https://github.com/opencontainers/runtime-spec/blob/v1.0.0/config.md#valid-values
+
+validation/linux_sysctl.t ........................... 20/26
+  Skipped: 6
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+
+validation/process_oom_score_adj.t .................. 20/26
+  Skipped: 6
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+
+validation/linux_devices.t .......................... 20/26
+  Skipped: 6
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+
+validation/process_capabilities_fail.t ......Any value which cannot be mapped to a relevant kernel interface MUST cause an error
+Refer to: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/config.md#linux-process
+validation/process_capabilities_fail.t .............. 20/27
+  not ok validation/process_capabilities_fail.t
+    timeout: 30000
+    file: validation/process_capabilities_fail.t
+    command: validation/process_capabilities_fail.t
+    args: []
+    stdio:
+      - 0
+      - pipe
+      - 2
+    cwd: /home/alban/go/src/github.com/opencontainers/runtime-tools
+    exitCode: 1
+
+  Skipped: 6
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+     syscall action SCMP_ACT_ALLOW
+
+validation/start.t .....exit status 2
+validation/start.t .................................... 6/7
+  not ok test count !== plan
+    +++ found                                                           
+    --- wanted                                                          
```

Comment:
`git log --check origin/pr/610` complains about trailing whitespace here (and in a few other places).  Can we strip that trailing whitespace after pasting in here, e.g. with `sed -i 's/[[:space:]]*$//' docs/runtime-compliance-testing.md`?